airbyte-workers/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java

```diff
@@ -57,6 +57,15 @@ public static NormalizationRunner create(String imageName, ProcessBuilderFactory
     }
   }
 
+  /**
+   * Check if destination configuration enabled the basic normalization feature
+   *
+   * @param config destination json config object
+   * @return the value of the basic normalization key
+   *
```

Comment:
this is for backward compatiblity and should go away once Subodh makes his change?

Reply:
yes